core/parsigdb/memory.go

```diff
@@ -92,10 +92,6 @@ func (db *MemDB) StoreExternal(ctx context.Context, duty core.Duty, signedSet co
 			continue
 		}
 
-		log.Debug(ctx, "Partial signed data stored",
```

Comment:
This log is annoying, generates a lot of entries, yet barely useful. Hence it was removed.

Reply:
Debugging is one time event, while it generates tons of traffic we pay for. I think we shall ensure we log sufficient warnings+errors to investigate in prod.

Reply:
We shouldn't run Debug mode in prod either way, right?
Can we at least move it to `StoreInternal`? This way we won't be logging on partial signatures received from peers and VC, but only from VC.
Another option is to move it to the more granular `Trace` which we currently aren't making use of.